great_expectations/rule_based_profiler/rule_based_profiler_result.py

```diff
@@ -17,13 +17,13 @@
 from great_expectations.types import SerializableDictDot
 
 
-@dataclass(frozen=True)
+@dataclass
```

Comment:
Is it necessary to get rid of immutability here?

Reply:
@billdirks I think that this may not be necessary any more, but I will check and if indeed so, fix in the next PR.  Thanks.